### PR TITLE
Update copy for first confirmation message

### PR DESCRIPTION
**Pull request by sniemeyer13**

### CODE REVIEW

File: spec/controllers/questions/consent_controller_spec.rb
Comment (bytheway875) [inline spec/controllers/questions/consent_controller_spec.rb:156]: there's been talk about us hopefully swapping out confirmation number from client_id to something else in the near future and/or next tax year. I think it could be a huge help to have a method on client called confirmation number that looks like this: 

``` 
def confirmation_number
  id
end
```
So that we can reference client.confirmation_number everywhere we need to, and then just swap that value out with the new confimation number and have it update everywhere. I have concerns that if we pass it in as client_id from the intake and id straight from the client sometimes, it will be much harder to track down all the places we're using it later on!

**Review by bytheway875 — COMMENTED** [review]
Good work! I will leave it up to you to decide whether creating an alias method called confirmation_code or some sort makes sense to do as part of this PR.

File: config/locales/en.yml
Comment (alaroia-cfa) [inline config/locales/en.yml:648]: We call these confirmation numbers elsewhere. Are you sure we're supposed to call them case numbers here? If so, that's fine, but thought I'd ask. Non-blocking.

File: config/locales/en.yml
Comment (alaroia-cfa) [inline config/locales/en.yml:652]: The footer links to support@getyourrefund.org . Are you sure this is the right email address? If so, great, but I wanted to check.

File: config/locales/en.yml
Comment (alaroia-cfa) [inline config/locales/en.yml:657]: I'm surprised we capitalize Your Tax Team like that here, when we don't capitalize it like that elsewhere in https://github.com/codeforamerica/vita-min/blob/3cf253221cffba978e9e2848c3bdffdd7fdd1679/config/locales/en.yml

Not a blocker, but wanted to raise it and see if you're sure!

File: config/locales/en.yml
Comment (alaroia-cfa) [inline config/locales/en.yml:660]: This says case number but we typically them confirmation numbers. See earlier note. If you're sure about this, fine with me!

File: config/locales/es.yml
Comment (alaroia-cfa) [inline config/locales/es.yml:655]: Same question about email address as in the English

File: config/locales/es.yml
Comment (alaroia-cfa) [inline config/locales/es.yml:664]: Same question about case number as in English

File: spec/controllers/questions/consent_controller_spec.rb
Comment (alaroia-cfa) [inline spec/controllers/questions/consent_controller_spec.rb:165]: Same question about capitalization of "Your tax team" vs. "Your Tax Team".

File: spec/controllers/questions/consent_controller_spec.rb
Comment (alaroia-cfa) [inline spec/controllers/questions/consent_controller_spec.rb:179]: Same Q about case # vs conf #

**Review by alaroia-cfa — APPROVED** [review]
Great work getting this copy changed. Getting the right info to clients is going to important!

If you can check with product about the copy concerns I had, that's great. If not, then let me know and I can follow up.

File: config/locales/en.yml
Comment (sniemeyer13) [reply config/locales/en.yml:657]: Confirmed with Anu - we will keep it capitalized.

File: config/locales/en.yml
Comment (sniemeyer13) [reply config/locales/en.yml:652]: Yes - seems that this link goes directly to their tax preparer so we want to use hello@

File: spec/controllers/questions/consent_controller_spec.rb
Comment (sniemeyer13) [reply spec/controllers/questions/consent_controller_spec.rb:156]: Ok! It seems like there's still some back and forth on what to call it/do with this so I'm going to hold off until we nail down the wording.

**Pull request merged**
